browser/resources/settings/email_aliases_page/content/email_aliases_modal.tsx

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Button from "@brave/leo/react/button"
+import Icon from "@brave/leo/react/icon"
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import * as React from 'react'
+import { ViewState, ViewMode, MappingService, MAX_ALIASES } from "./types"
+import { Col, Row } from "./style"
+import Input, { InputEventDetail } from "@brave/leo/react/input"
+import styled from "styled-components"
+
+const Modal = styled(Col)`
+  border-radius: var(--cr-card-border-radius);
+  background-color: white;
+  z-index: 2;
+  border: none;
+  opacity: 100%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%,-50%);
+  justify-content: flex-start;
+`
```

Comment:
rather than creating your own modal can you use the HTML dialog element (or even better, the Nala modal if it works)?

Reply:
done